website/docs/installation/documentdb/deb.md

```diff
@@ -13,6 +13,37 @@ We provide different DocumentDB `.deb` packages for various deployments on our [
   It includes features that significantly slow down performance and is not recommended for production use.
 
 Download the appropriate DocumentDB `.deb` package from our release page.
-Then, you can use `dpkg` tool to install it.
 
-You need to install PostgreSQL and additional dependencies required by the DocumentDB extension.
+Before installing the DocumentDB extension, you need to install PostgreSQL and additional dependencies required by the DocumentDB extension.
+
+With the dependencies installed, you can install the DocumentDB extension using `dpkg`.
+For example, to install the `documentdb.deb` package, run the following command:
+
+```sh
+sudo dpkg -i /path/to/documentdb.deb
+```
+
+Ensure to replace `/path/to/documentdb.deb` with the actual path and filename of the downloaded `.deb` package.
+
+Once installed, update your `postgresql.conf` to load the extension libraries on startup into the default `postgres` database.
+Add the following lines to `postgresql.conf`:
+
+```text
+shared_preload_libraries = 'pg_cron,pg_documentdb_core,pg_documentdb'
+cron.database_name       = 'postgres'
+```
+
+Then create the extension by running the following SQL command within the `postgres` database:
+
+```sql
+CREATE EXTENSION documentdb CASCADE;
+```
+
+:::caution
+We recommend using the default `postgres` database; some features may not work as expected if you use a different database.
```

Comment:
Do we know what features may not work?

Reply:
Not all, but if this is not updated for example, `cron.database_name       = 'postgres'`, the extensions won't be installed. And there may be other instances where this comes into play. I've tested it using a different database, and while things are okay at a glance, I can't say conclusively that everything works as expected.

Reply:
> I can't say conclusively that everything works as expected.

Then, we should not recommend that, and that should not be an option.

We also don't have any documentation about using non-admin users; there is an open issue about things not working with a regular user. And our code and docs should be in sync.